app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -26,7 +26,7 @@ public function transform(Post $post)
                 'uri' => $post->getMediaUrl(),
                 'type' => 'image',
             ],
-            'tagged' => $post->tagNames(),
+            'tagged' => $post->tags()->pluck('tag_names'),
```

Comment:
Ugh I kept trying to search all files for `tags()->pluck('tag_names')` in sublime text but it was saying there were none! Good find!!!